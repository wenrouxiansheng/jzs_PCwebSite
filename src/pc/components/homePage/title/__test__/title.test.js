import React from "react";
import renderer from "react-test-renderer";
import Title from "../index";

test("PC:title 组件测试", () => {
  const component = renderer.create(
    <Title
      info={{
        title: "为什么选择桔子树",
        subTitle: "Core advantages",
      }}
      style={{
        color: "red",
      }}
    />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

  // 手动触发回调
  // tree.props.onMouseEnter();
  // 重新渲染
  tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
