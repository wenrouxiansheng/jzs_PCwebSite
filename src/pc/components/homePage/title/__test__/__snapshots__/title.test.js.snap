// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PC:title 组件测试 1`] = `
<div
  className="animationTitle"
  style={
    Object {
      "color": "red",
    }
  }
>
  <h3>
    <b>
      为什么选择桔子树
    </b>
  </h3>
  <h6>
    Core advantages
  </h6>
</div>
`;

exports[`PC:title 组件测试 2`] = `
<div
  className="animationTitle"
  style={
    Object {
      "color": "red",
    }
  }
>
  <h3>
    <b>
      为什么选择桔子树
    </b>
  </h3>
  <h6>
    Core advantages
  </h6>
</div>
`;

exports[`title 组件测试 1`] = `
<div
  className="animationTitle"
  style={
    Object {
      "color": "red",
    }
  }
>
  <h3>
    <b>
      为什么选择桔子树
    </b>
  </h3>
  <h6>
    Core advantages
  </h6>
</div>
`;

exports[`title 组件测试 2`] = `
<div
  className="animationTitle"
  style={
    Object {
      "color": "red",
    }
  }
>
  <h3>
    <b>
      为什么选择桔子树
    </b>
  </h3>
  <h6>
    Core advantages
  </h6>
</div>
`;
